README.md

```diff
@@ -80,13 +81,20 @@ OPTIONS
 
             Coalesce 'runs-on' labels by a first word
 
+    --repo-path
+    
+            owner/repository
+            Uses in `fetch.py` result files path, so `last_seen.py` needs it to
+            find data for certain repo. Default: 'tarantool/tarantool'. You can
+            set only one repo in one sckript start.
+
 EXAMPLE
 
     Generate a report in the HTML format for branches `master` and
-    `sample-branch`:
+    `sample-branch` for repo 'tarantool/multivac':
 
 ```console
-$ ./multivac/last_seen.py --branch master --branch sample-branch --format html
+$ ./multivac/last_seen.py --branch master --branch sample-branch --format html --repo-path tarantool/multivac
```

Comment:
why `tarantool/multivac`?

Reply:
That's README in tarantool/multivac repo. To gather data from tarantool/tarantool we don't need to set it as CLI argument because it's default value.

Reply:
ok